diff --git a/mkman.sh b/mkman.sh
index 79ad162..3c4260b 100755
--- a/mkman.sh
+++ b/mkman.sh
@@ -5,8 +5,7 @@
 
 set -e
 
-sed "s/VERSION/$1/g" "$2" > "$3"
-MONTH="$(LC_ALL=C date -u --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%b)"
-sed -i "s/MONTH/$MONTH/g" "$3"
-YEAR="$(LC_ALL=C date -u --date="@${SOURCE_DATE_EPOCH:-$(date +%s)}" +%Y)"
-sed -i "s/YEAR/$YEAR/g" "$3"
+sed -e "s/VERSION/$1/g" \
+	-e "s/MONTH/$(date -u +%b)/g" \
+	-e "s/YEAR/$(date -u +%Y)/g" \
+	"$2" > "$3"
